api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCycleServiceTest.java

```diff
@@ -48,11 +50,11 @@ void shouldCreateNewPaymentCycle() {
     }
 
     @Test
-    void shouldCreateNewPaymentCycleForEligibleClaim() {
+    void shouldCreateNewPaymentCycleWithExpectedDueDateForEligibleClaimWithPregnancyVouchers() {
         LocalDate today = LocalDate.now();
-        Claim claim = aValidClaim();
+        Claim claim = aClaimWithExpectedDeliveryDate(today);
         List<LocalDate> datesOfBirth = List.of(LocalDate.now(), LocalDate.now().minusDays(2));
```

Comment:
I know this is only a test and technically makes no difference, but having a Claimant with an EDD of today and two children whose dates of birth are today and two days ago is hardly realistic, can we correct this please?

Reply:
I've changed the EED to nine months in the future.